### DISCUSSION TITLE
Show if Windows Updates are pending a restart

### DISCUSSION THREAD

**Discussion by DeFlanko** [timmo001/system-bridge#2790] [Ideas] [answered]
Is there a way to report if the Windows OS is pending a restart due to applied Windows updates so that information can be shown in Home Assistant?

**Comment by fuatakgun** [comment]
Maybe, you can run a PowerShell script to find out.

https://www.geekbits.io/how-to-view-a-pending-reboot-in-windows/

**Reply by timmo001** [reply]
Thanks for this article, I will take a look at using it. Since the application is meant to be multiplatform, I will also look if there is a similar command other OSs

**Reply by timmo001** [reply, marked as answer]
#2844

**Comment by DeFlanko** [comment]
The question was more geared is System-bridge could do it. Im well aware of how other tools can report this info, id like to see if there is a way to integrate this functionality into system-bridge so that it can be viewed on Home Assistant.
